source/2020-04-03-the-ember-times-issue-142.md

```diff
@@ -61,14 +62,14 @@ Try it out today by ember installing [ember-arg-types](https://github.com/jkusa/
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Betas of Glimmer.js 2 🎉](https://github.com/glimmerjs/glimmer.js/releases)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+A set of **new betas for Glimmer.js 2.0.0** ([beta.1](https://github.com/glimmerjs/glimmer.js/releases/tag/v2.0.0-beta.1) && [beta.3](https://github.com/glimmerjs/glimmer.js/releases/tag/v2.0.0-beta.3)) have been released! 🔥🔥🔥 
+
+The releases incorporate some exciting new developments including a new minimal `renderComponent` API, a template import based design, a standardized build based on plain JavaScript modules, a new Glimmer.js app blueprint and support for custom helpers and modifiers. The standardized build unlocks the ability to use whatever custom build pipeline you might want (WebPack, Parcel, Rollup, etc.)!
+
+The Glimmer.js betas could pave the way for what we might want to do in Ember.js, or even what we might _not_ want to do in Ember.js. With that in mind, please note that these new Glimmer.js betas are definitely experimental 🔬🧪 and should while have fun exploring them 🤠, but also keep that in mind when considering them for use in your application.
```

Comment:
I wasn't sure what you meant by "and should while", so I omitted it in the suggestion below. Can you check to see if it matches your intent?
```suggestion
The Glimmer.js betas could pave the way for what we might want to do in Ember.js, and even what we might _not_ want to do in Ember.js. Please note that these new Glimmer.js betas are definitely experimental 🔬🧪Have fun exploring them 🤠, but keep this in mind when considering them for use in your application.
```